Frameworks/Foundation/NSURLSession.mm

```diff
@@ -761,15 +757,21 @@ - (void)downloadTask:(NSURLSessionDownloadTask*)task
                              totalBytesExpectedToWrite);
 }
 
+- (void)_destroyTemporaryFile:(NSURL*)url {
+    [[NSFileManager defaultManager] removeItemAtURL:url error:nullptr];
```

Comment:
yes, nil

---
In reply to: [95426628](https://github.com/Microsoft/WinObjC/pull/1670#discussion_r95426628) [](ancestors = 95426628)